docs/guides-faq.md

```diff
@@ -73,7 +71,7 @@ You can also define a `~/.aragon/mnemonic.json` file like:
 
 ## Verifying your contracts
 
-To be sure you are trying to verify the correct smart contract be sure to check what app version is installed in you DAO with:
+You can validate the smart contract and other files of a specific app installed in your DAO with:
```

Comment:
```suggestion
You can validate the smart contract and other files of a specific app installed in your Aragon organization with:
```

Typically we try to avoid "DAO" as much as possible (if you see this in other places, please continue rewording them to "organization" or "Aragon organization"!).